core/lib/eth_client/src/clients/http/signing.rs

```diff
@@ -35,11 +35,44 @@ impl PKSigningClient {
     ) -> Self {
         // Gather required data from the config.
         // It's done explicitly to simplify getting rid of this function later.
-        let main_node_url = &eth_client.web3_url;
         let operator_private_key = eth_sender
             .sender
             .private_key()
             .expect("Operator private key is required for signing client");
+
+        Self::from_config_inner(
+            eth_sender,
+            contracts_config,
+            eth_client,
+            operator_private_key,
+        )
+    }
+
+    /// Create an signing client for the blobs account
+    pub fn from_config_blobs(
+        eth_sender: &ETHSenderConfig,
+        contracts_config: &ContractsConfig,
+        eth_client: &ETHClientConfig,
+    ) -> Option<Self> {
+        // Gather required data from the config.
+        // It's done explicitly to simplify getting rid of this function later.
```

Comment:
?? do we want to remove this function later?

Reply:
It is a copy of the function above with another name; comment kept for unity.